packages/deprecation-crawler/src/lib/crawler/index.ts

```diff
@@ -27,6 +25,10 @@ export async function crawlDeprecations(
   const deprecations = sourceFiles
     // TODO: seems like these files cannot be parsed correctly?
     .filter((file) => !file.getFilePath().includes('/Observable.ts'))
+    .filter((file) => {
```

Comment:
> I thought about where to implement this logic. Is it a good idea to take if from config/cli and add it to the includes section of our tsConfig file? Will the matching files still get included? Also, you may have other opinions on that case...

I think it should overwrite the default config (for this run), instead of including it.
The change can be limited to https://github.com/ts-code-health/deprecation-manager/blob/master/packages/deprecation-crawler/src/lib/utils.ts#L320 imho. We can create a tsconfig with only the includes from the path filter